### ISSUE TITLE
account suspended 

### ISSUE THREAD

**Issue by Turhvjbufv** [KhoalaS/Infinity-For-Reddit#5]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.8.0

### Where did you get the app from

Other

### Android version

Android 12 

### First occurred

today

### Steps to reproduce

Used the app for a few days, after that my account got suspended might wanna check this out

**Comment by ixfd64** [comment]
Did you get any messages saying why you were suspended?

**Comment by Turhvjbufv** [comment]
Breaking content policy

**Comment by KhoalaS** [comment]
Since it is not a ban for breaking the User Agreement I don't think it is because of the app. Could you post the whole message you got from Reddit? (with Personal Info etc. redacted of course)

**Comment by Turhvjbufv** [comment]
![Reddit.png](https://github.com/KhoalaS/Infinity-For-Reddit/assets/91851160/c60d7260-c9bb-4f46-8c0c-bb51f20c8005)

**Comment by Turhvjbufv** [comment]
You are right never mind

**Comment by ConfusedDiscord** [comment]
The same thing just happened to me (I only used my alt on the apk that allows us to input our own api keys). I received the same exact message. I hadn't made any comments or posts/upvoted anything on the account really. the account was verified and was created over a year ago. I'm not saying me using the app is definitely the cause of this, just wanted to add my experience here. As for me it doesn't bother me since I only browsed subs on that acc anyway, and I'm still able to. Just hoping nothing bad will happen to my main. (I only used the official reddit app for that one). I'll try to contact reddit anyway to see what they have to say and if this can even get resolved.
